lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _get = require('lodash.get');
+
+exports.register = (defaultRate, routes) => {
+  const rateTable = {};
+
+  routes.forEach((route) => {
+    if (!(route.method === 'get' || route.method === 'post' || route.method === 'delete')) {
+      return;
+    }
+
+    if (_get(route, 'settings.plugins.rateLimit.enabled') === false) {
```

Comment:
Since you use `route.settings.plugins` here and down below, I think you can just move the `const plugins = route.settings.plugins;` line to the top of the function and use that throughout. You couldn't do that before since it was two functions (`filter` and `forEach`), but now you can. I think then you can even get rid of the `lodash.get` at that point.